### ISSUE TITLE
why can't android watch-only wallet send bch?

### ISSUE THREAD

**Issue by amo98** [Electron-Cash/Electron-Cash#2373]
I want to use two android phone to set up a cool wallet. One have private key and the other have a watch-only address. But it can not be done. Because android watch-only wallet can not send bch. But linux watch-only wallet can send , then I can sign with my android cool wallet! 
And other question is , watch-only wallet can only watch ONE address. If I have more address to watch, I have to set up some wallets but not just one wallet!
Please help!

**Comment by mhsmith** [comment]
> android watch-only wallet can not send bch. But linux watch-only wallet can send , then I can sign with my android cool wallet!

No watch-only wallet can *send* a transaction. Do you mean you want the watch-only wallet to create an unsigned transaction which you then sign and send with a different wallet which has the private key? Why not just create the transaction with the second wallet?

> watch-only wallet can only watch ONE address.

Actually you can provide multiple addresses when creating the wallet. But unlike the desktop app, the Android app can't add more addresses to the wallet later.

**Comment by amo98** [comment]
> > android watch-only wallet can not send bch. But linux watch-only wallet can send , then I can sign with my android cool wallet!
> 
> No watch-only wallet can _send_ a transaction. Do you mean you want the watch-only wallet to create an unsigned transaction which you then sign and send with a different wallet which has the private key? Why not just create the transaction with the second wallet?
Yes, I want to create an unsigned transaction and then sign with cool wallet.The cool wallet has no input transaction then it can not create transaction. PC (windows, linux) watch-only wallet can create an unsigned transaction. BUT android watch-only wallet just can't !
> 
> > watch-only wallet can only watch ONE address.
> 
> Actually you can provide multiple addresses when creating the wallet. But unlike the desktop app, the Android app can't add more addresses to the wallet later.
Oh,yeah! Thanks!

**Comment by mhsmith** [comment]
OK, I think if your other wallet has the private key then it should be able to create the transaction all by itself. But I agree this could be a useful feature, so I'll leave the issue open in case anyone else wants it.

**Comment by KyrosKrane** [comment]
I would consider this a valuable feature. It's already viable with two desktop clients, as they have the functionality to create unsigned transactions, export transactions (signed or unsigned) as QR codes, and scan to import transactions from QR codes. It would be much handier to have the watch-only wallet be on a mobile device. So, in that vein, I would request this for both iPhone and Android clients.
